.github/workflows/tests.yml

```diff
@@ -0,0 +1,27 @@
+name: Tests
+on: 
+  repository_dispatch:
+    types: [tests-report]
+  push:
+jobs:
+  tests:
+    strategy:
+      matrix:
+        platform: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.platform }}
+    env:
+      RUST_BACKTRACE: 1
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v2
+      - name: Install Node
+        uses: actions/setup-node@v2
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable-2020-10-08
+          target: wasm32-unknown-unknown
+      - name: Run unit tests
+        run: cd scripts && chmod +x unit_tests.sh && ./unit_tests.sh
```

Comment:
You should be able to `chmod` a script on your PC and push this change.

Reply:
Have moved scripts to the yaml file will update README and PR description accordingly. Thank you!